spec/lib/phobos_spec.rb

```diff
@@ -28,4 +28,18 @@
       expect(Phobos.create_exponential_backoff).to be_a(ExponentialBackoff)
     end
   end
+
+  describe '.logger' do
+    context 'with "config.logger.file" defined' do
```

Comment:
Maybe good to have a test for the case when the `file` is not defined or is set to `false`. Something like:
```ruby
file_configs = [ false, nil ]
file_config.each do |file_config|
  Phobos.config.logger.file = file_config
  expect { Phobos.configure_logger }.to_not raise_error(SomeError)
end
```

WDYT?